### PR TITLE
add bundle option to module-overview

### PR DESCRIPTION
**Pull request by BoDonkey**
## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*

The `module-overview` reference page does not document the `bundle` option. This PR adds that option. Closes PRO-2293.

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

1. Build the documentation
2. Navigate to `/reference/module-api/module-overview.html#configuration-settings`

## What kind of change does this PR introduce?
*(Check at least one)*

- [ ] Bug fix
- [ ] New feature
- [ ] Refactor
- [X] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [X] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [ ] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: docs/reference/module-api/module-overview.md [hunk 1]
```diff
@@ -98,6 +99,10 @@ An object used to add additional, often optional, settings to a module. There ar
 
 Set to `false` to prevent the module from being fully instantiated in the application. The primary purpose of this option is to create a base class module that others will [extend](#extend) but that will not be used directly.
 
+### `bundle`
+
+Used to add multiple modules from a single base module. Takes an object with two properties. The `directory` property takes a relative path to the directory of the modules to be loaded. The `modules` property takes an array of module names. Modules loaded in this way still need to be added to the `app.js` file, unless they use [`improve`](#improve). Any modules that to be used as a base class for other modules should be added to `app.js`, but have their [`instantiate`](#instantiate) option set to `false`.
```
Comment (boutell) [inline docs/reference/module-api/module-overview.md:104]: "from a single npm module." It must be an npm module and has no effect at all in a project level module, so let's make that clear from the get-go.

Comment (boutell) [reply docs/reference/module-api/module-overview.md:104]: "instantiate property", not "instantiate option" (it's not an option, it doesn't go under `options`).

Comment (boutell) [reply docs/reference/module-api/module-overview.md:104]: The original module must be activated in `app.js` as well if any of the other modules in the bundle are to be used. The original module can and usually does contain functionality of its own, in addition to delivering a bundle of additional modules.

**Review by boutell — CHANGES_REQUESTED** [review]

File: docs/reference/module-api/module-overview.md (hunk 1 above)
Comment (BoDonkey) [reply docs/reference/module-api/module-overview.md:104]: Altered test for all three comments.

File: docs/reference/module-api/module-overview.md [hunk 2]
```diff
@@ -101,7 +101,7 @@ Set to `false` to prevent the module from being fully instantiated in the applic
 
 ### `bundle`
 
-Used to add multiple modules from a single base module. Takes an object with two properties. The `directory` property takes a relative path to the directory of the modules to be loaded. The `modules` property takes an array of module names. Modules loaded in this way still need to be added to the `app.js` file, unless they use [`improve`](#improve). Any modules that to be used as a base class for other modules should be added to `app.js`, but have their [`instantiate`](#instantiate) option set to `false`.
+Used to add multiple modules from a single npm module. Takes an object with two properties. The `directory` property takes a relative path to the directory of the modules to be loaded. The `modules` property takes an array of module names. THe original module and bundled modules loaded in this way still need to be added to the `app.js` file, unless they use [`improve`](#improve). Any modules that to be used only as a base class for other modules should be added to `app.js`, but have their [`instantiate`](#instantiate) property set to `false`.
```
Comment (boutell) [inline docs/reference/module-api/module-overview.md:104]: that to be used only -> that are to be used only

**Review by boutell — CHANGES_REQUESTED** [review]

File: docs/reference/module-api/module-overview.md (hunk 2 above)
Comment (BoDonkey) [reply docs/reference/module-api/module-overview.md:104]: Damn. Changed.

**Review by boutell — APPROVED** [review]

**Pull request merged**